models/articles.models.js

```diff
@@ -12,4 +12,21 @@ function fetchArticleById(article_id){
     })
 }
 
-module.exports = {fetchArticleById}
\ No newline at end of file
+function fetchAllArticles(){
+    const queryString = `SELECT a.article_id,a.title,a.topic,a.author, CAST(a.created_at AS DATE),a.votes,a.article_img_url,CAST(COUNT(c.article_id)AS INT) AS "comment_count"
+    FROM articles AS a 
+    JOIN comments AS c
```

Comment:
Consider a different type of JOIN if you don't want to omit articles that don't have comments from the response.